docs/projects/coordinating-multiple-projects/deploy-release-step/index.md

```diff
@@ -1,39 +1,51 @@
 ---
-title: Deploy release step
-description: The deploy release step allows you to trigger the deployment of a release of a project from another project
+title: Deploy a release step
+description: The deploy a release step allows you to trigger the deployment of a release of a project from another project
 ---
 
-The _Deploy Release_ step lets you have a project trigger the deployment of a release of another project. This is useful when you are [coordinating multiple projects](/docs/projects/coordinating-multiple-projects/index.md).
+The _Deploy a Release_ step lets you have a project trigger the deployment of a release of another project. This is useful when you are [coordinating multiple projects](/docs/projects/coordinating-multiple-projects/index.md).
 
 ![Deploy release step card](deploy-release-card.png "width=500")
 
-when you add a _Deploy Release_ step to your deployment process, you can then select the project which will be deployed.
+when you add a _Deploy a Release_ step to your deployment process, you can then select the project which will be deployed.
 
 ![Deploy release select project](deploy-release-step-select-project.png "width=500")
 
-You can add many _Deploy Release_ steps to your process, if you wish to deploy releases of many projects.
+You can add many _Deploy a Release_ steps to your process, if you wish to deploy releases of many projects.
 
-When creating a release of a project containing _Deploy Release_ steps you can select the release version of each project, similar to the way versions of packages are selected.  
+## Creating a release
+
+When creating a release of a project containing _Deploy a Release_ steps you can select the release version of each project, similar to the way versions of packages are selected:
+
+![Create release with deploy release steps](deploy-release-create-release-screen.png "width=500")
+
+### Channels
+
+The [Channel](/docs/releases/channels/index.md) used for any _Deploy a Release_ step is automatically determined by the release version of the project you select in the create release screen, since a channel is chosen when a release is created.
+
+:::hint
+It's possible to have the _Deploy a Release_ step pick the channel for the child project you want it to use using [package version rules](/docs/releases/channels/index.md#Channels-versionrules). Watch our - [Ask Octopus Episode: Deployment Channels with Child Projects](https://www.youtube.com/watch?v=3oLVq1EpUfc) to see it in action.
```

Comment:
I think this could be worded better but I'm having trouble getting it...

It's possible to choose child releases in specific channels for the Deploy a Release step using package version rules. ?

Reply:
Have tweaked slightly in 4ece90bb9bd8dd545a487ae728ef090518af0999 to:

> It's possible to choose child releases from specific channels when using the _Deploy a Release_ step using package version rules ...